### PR TITLE
Contribuciones

### LINKED ISSUE #8: add glosario

### PR DESCRIPTION
**Pull request by Joskerus**
Fix #8 

* **Please check if the PR fulfills these requirements**

- [ ] I have read the CONTRIBUTING guidelines
- [ ] A new item has been added to `NEWS.md`
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Checks have been run locally and pass

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by Joskerus** [comment]
Cambios a las unidades y glosario

File: learners/reference.md [hunk 1]
```diff
@@ -1,7 +1,1292 @@
...
+|                                | **Enf e rmos** | **S anos** | **T otal** |
++--------------------------------+----------------+------------+------------+
+| **Estu d iantes pri m aria**   | 30             | 120        | 150        |
++--------------------------------+----------------+------------+------------+
+| **Estu d iantes secun d aria** | 5              | 135        | 140        |
++--------------------------------+----------------+------------+------------+
+| -   \*To t                     | 35             | 265        | 290        |
+|                                |                |            |            |
+| al\*\*                         |                |            |            |
++--------------------------------+----------------+------------+------------+
```
Comment (avallecam) [inline learners/reference.md:732]: with https://www.tablesgenerator.com/markdown_tables# we build a fix tables for markdown

```suggestion
|  | Enfermos | Sanos | Total |
|---|---|---|---|
| **Estudiantes primaria** | 30 | 120 | 150 |
| **Estudiantes secondaria** | 5 | 135 | 140 |
| Total | 35 | 265 | 290 |
```

File: learners/reference.md [hunk 2]
```diff
@@ -1,7 +1,1292 @@
...
+|                           |                  |     oles)\*\* |           |
++---------------------------+------------------+---------------+-----------+
+| **Ex puestos\             | 35               | 25            | 60        |
+| (Consumió S andwich)**    |                  |               |           |
++---------------------------+------------------+---------------+-----------+
+| **No expuestos\           | 15               | 25            | 40        |
+| (No Consumió S andwich)** |                  |               |           |
++---------------------------+------------------+---------------+-----------+
+| **Total**                 | 50               | 50            | 100       |
++---------------------------+------------------+---------------+-----------+
```
Comment (avallecam) [inline learners/reference.md:800]:
```suggestion
|  | Enfermos (Casos) | Sanos (Controles) | Total |
|---|---|---|---|
| **Expuestos (Consumió Sandwish)** | 35 | 25 | 60 |
| **No expuestos (No consumió Sandwish)** | 15 | 25 | 40 |
| Total | 50 | 50 | 100 |
```

**Review by avallecam — CHANGES_REQUESTED** [review]
Gracias por la actualizacion! Solo solicito que aceptes los cambios del formato de tablas en markdown, porque se construyen con espacios entre palabras. Una herramienta que funciona bien es https://www.tablesgenerator.com/markdown_tables

**Comment by avallecam** [comment]
listo para el merge 🚀 

en una actualizacion aparte podemos convertir el glosario en Rmd para que pueda alojar el codigo de la imagen que compara R0 #11

**Pull request merged**